### DISCUSSION TITLE
Profile Description Doesn't Preserve Newlines

### DISCUSSION THREAD

**Discussion by coreyja** [BattlesnakeOfficial/feedback#223] [Bug Reports]
Maybe not technically a bug, but I think it would be cool if the new lines in your profile were preserved when it gets displayed. It looks like the DB values have the newlines, its just that css is collapsing the whitespace when it renders

I was able to 'fix' this locally by adding `white-space: pre-wrap` to the `p` tag that contains the description!

Before:
![Screenshot 2023-01-17 at 9 59 22 PM](https://user-images.githubusercontent.com/964771/213072289-21c1a06e-b53d-4403-90d8-4aadf7641966.png)

After:
![Screenshot 2023-01-17 at 10 00 06 PM](https://user-images.githubusercontent.com/964771/213072301-70e7aa16-fc88-42aa-b4f3-ea1f5d4766e7.png)

**Comment by originalwebgurl** [comment]
Thanks Corey! I'll add this asap. It should roll out sometime tomorrow.

**Reply by coreyja** [reply]
Awesome thanks!!

**Reply by originalwebgurl** [reply]
The fix was released yesterday. Thanks again for letting us know.
